Paintroid/src/main/res/values-af-rZA/string.xml

```diff
@@ -70,7 +70,6 @@
     <string name="menu_save_copy">Save copy</string>
     <string name="menu_hide_menu">Fullscreen</string>
     <string name="menu_rate_us">Rate us!</string>
-    <string name="menu_try_pocket_code">Try Pocket Code!</string>
```

Comment:
only change strings in `Paintroid/src/main/res/values/string.xml `. all other string files will be updated by crowdin